server/superdesk/data_initialization/import.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
It is just that it's a bit more work. This is on the plan just that I added this if we need a way to bootstrap a new env quickly.